Spanner/src/Connection/Grpc.php

```diff
@@ -476,6 +476,9 @@ public function createBackup(array $args)
     {
         $backup = $this->pluck('backup', $args);
         $backup['expireTime'] = $this->formatTimestampForApi($this->pluck('expireTime', $backup));
+        if (isset($args['versionTime'])) {
```

Comment:
Should we also check here that versionTime is of `DateTimeInterface` or is that taken care of by the `formatTimestampForApi` function?

Reply:
`formatTimestampForApi()` returns `array[seconds, nanos]`, which is compatible with `google.protobuf.Timestamp`. :)